### ISSUE TITLE
feat: Add styling functions to Canvas Kit Styling

### ISSUE THREAD

**Issue by alanbsmith** [Workday/canvas-kit#2440]
## 🚀 Feature Proposal

Add helpful CSS styling functions to `canvas-kit-styling`.

- `negate` - interpolate `calc($VAR * -1)`
- Happy to add other functions here too (add, subtract, etc).

## Motivation

Doing basic variable manipulation is tedious with CSS vars. Helper functions would make that nicer.

## Example

```ts
import { cssVar, cssNegate, createStyles } from '@workday/canvas-kit-styling';

const styles = createStyles({
  position: 'absolute',
  top: cssNegate(cssVar(system.space.x4)),
});
```

**Comment by alanbsmith** [comment]
Polished has a `math` function that's handy: https://polished.js.org/docs/#math

**Comment by NicholasBoll** [comment]
Closed by #2475
